src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -129,12 +129,16 @@ private function validate(string $content, string $file = null): array
         $document->loadXML($content);
 
         if (null !== $targetLanguage = $this->getTargetLanguageFromFile($document)) {
-            $normalizedLocale = preg_quote(str_replace('-', '_', $targetLanguage), '/');
+            $normalizedLocalePattern = sprintf(
+                '(%s|%s)',
+                $targetLanguage,
```

Comment:
missing preg_quote about, isn't it?
can you also please keep the call on one line?

Reply:
@nicolas-grekas 

Adjusted!